packages/core/event.ts

```diff
@@ -17,6 +17,9 @@ export function getEventName(eventNameObj: EventNameObjectType, forAttendeeView
       eventName: eventNameObj.eventType,
       host: eventNameObj.host,
       attendeeName: eventNameObj.attendeeName,
+      interpolation: {
```

Comment:
as per https://github.com/calcom/cal.com/issues/7255#issuecomment-1442027857 . shout out to @CarinaWolli